lib/utils/keyboard_manager.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class KeyboardManager {
+  static dismiss(BuildContext context) {
```

Comment:
It would be better if we can put this function into `build_context_ext` with the named `dismissKeyboard`. What do you think? 🤔

Reply:
Good suggestion 👍 Updated in 4844a0c.